Formula/k/kubevpn.rb

```diff
@@ -0,0 +1,36 @@
+class Kubevpn < Formula
+  desc "Offers a Cloud-Native Dev Environment that connects to your K8s cluster network"
+  homepage "https://www.kubevpn.cn"
+  url "https://github.com/kubenetworks/kubevpn.git",
+      tag:      "v2.2.11",
+      revision: "98d88ac542c06e8f66937e7f57b5ceae514b0c30"
+  license "MIT"
+  head "https://github.com/kubenetworks/kubevpn.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    goos = Utils.safe_popen_read("#{Formula["go"].bin}/go", "env", "GOOS").chomp
+    goarch = Utils.safe_popen_read("#{Formula["go"].bin}/go", "env", "GOARCH").chomp
+    tags = "noassets"
+    project = "github.com/wencaiwulue/kubevpn/v2"
+    ldflags = %W[
+      -s -w
+      -X #{project}/pkg/config.Image=docker.io/naison/kubevpn:v#{version}
+      -X #{project}/pkg/config.Version=v#{version}
+      -X #{project}/pkg/config.GitCommit=#{Utils.git_head}
+      -X #{project}/cmd/kubevpn/cmds.BuildTime=#{time.iso8601}
+      -X #{project}/cmd/kubevpn/cmds.Branch=#{Utils.git_branch}
+      -X #{project}/cmd/kubevpn/cmds.OsArch=#{goos}/#{goarch}
+    ]
+    system "go", "build", *std_go_args(ldflags:, output: bin/"kubevpn"), "-tags", tags, "./cmd/kubevpn"
+
+    generate_completions_from_executable(bin/"kubevpn", "completion")
+  end
+
+  test do
+    assert_includes shell_output("#{bin}/kubevpn version"), "Version: v#{version}"
```

Comment:
```suggestion
    assert_match "Version: v#{version}", shell_output("#{bin}/kubevpn version")
```

Reply:
```shell
➜  bin kubevpn version
KubeVPN: CLI
    Version: v2.2.11
    DaemonVersion: v2.2.11
    Image: docker.io/naison/kubevpn:v2.2.11
    Branch: HEAD
    Git commit: 98d88ac542c06e8f66937e7f57b5ceae514b0c30
    Built time: 2024-06-24 16:30:04
    Built OS/Arch: darwin/arm64
    Built Go version: go1.22.4
➜  bin
```
@SMillerDev because version output is like this, so use include to judge

Reply:
Yes, but `includes` is for object and `match` is for strings.

Reply:
So we have method to judge substring for strings

Reply:
Yes, assert_match. That's why I suggested it.

Reply:
assert_match is for all string match? but we needs substring. one string contains another substring

Reply:
> Yes, assert_match. That's why I suggested it.

I tested it, looks works fine 😂, i will change it

Reply:
Already changed it